os/android/iahwc2.cpp

```diff
@@ -682,8 +682,6 @@ HWC2::Error IAHWC2::HwcDisplay::PresentDisplay(int32_t *retire_fence) {
     layers.emplace_back(l.second->GetLayer());
   }
 
-  if (layers.empty())
```

Comment:
When layers is empty, will return in PhysicalDisplay::Present.
If return in HWC2Display when layers is empty,  the LogicalDisplayManager can't do the present at right moment. 
This will cause Composer process to crash  when using LogicalDisplay.

Reply:
the pointer "display_" not only refer to physical display, also virtual, virtual panorama, logical and mosaic display. it will make other display handle empty layers.